### ISSUE TITLE
Requirements.txt is out of date

### ISSUE THREAD

**Issue by robertklee** [yacoubb/stock-trading-ml#14]
Tensorflow and keras download the wrong version. I will open a PR to fix this issue.

**Comment by XyzzyX44** [comment]
You can update the requirements text to include for example:
tensorflow==1.12.0
to get the versions you want to run.

**Comment by robertklee** [comment]
I opened a PR #15 that fixes this. If you can merge that would be great

**Comment by pcazzola** [comment]
It seems to run with tensorflow 2.3 if you make the following changes:

1) move tensorflow up in the requirements.txt to be 3rd (after pandas) so that it installs a compatible version of numpy
2) Update the called for setting the random seed

Change this:
from tensorflow import set_random_seed
set_random_seed(4)

To this:
tf.random.set_seed(4)

3) Add this fix
https://github.com/yacoubb/stock-trading-ml/issues/18
